.github/workflows/go.yml

```diff
@@ -0,0 +1,58 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    name: Build + Test Stable
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: auto
+    steps:
+
+    - name: Build
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.16.x'
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Get Coverage
+      run:  go get golang.org/x/tools/cmd/cover
+
+    - name: Build
+      run: go build -v ./...
+
+    - name: Test
+      run: go test ./... -race -coverprofile=coverage.txt -covermode=atomic
+
+    - name: Coverage
+      run: bash <(curl -s https://codecov.io/bash)
```

Comment:
the Bash Uploader is deprecated because of [its security incident](https://about.codecov.io/security-update/)

https://docs.codecov.com/docs/about-the-codecov-bash-uploader
> Support for the Bash Uploader will be deprecated on February 1st, 2022.

I recommend to use [Codecov GitHub Action](https://github.com/marketplace/actions/codecov)

Reply:
Thanks! I've replaced this. Just a trivial FYI, the codecov action is still using this same coverage report service, just the API has changed.